homeassistant/components/sensor/amcrest.py

```diff
@@ -4,92 +4,51 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/sensor.amcrest/
 """
+import asyncio
 from datetime import timedelta
 import logging
 
-import voluptuous as vol
-import homeassistant.helpers.config_validation as cv
-
-from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.const import (
-    CONF_HOST, CONF_NAME, CONF_MONITORED_CONDITIONS,
-    CONF_USERNAME, CONF_PASSWORD, CONF_PORT, STATE_UNKNOWN)
+from homeassistant.components.amcrest import SENSORS
 from homeassistant.helpers.entity import Entity
-import homeassistant.loader as loader
-
-from requests.exceptions import HTTPError, ConnectTimeout
+from homeassistant.const import STATE_UNKNOWN
 
-REQUIREMENTS = ['amcrest==1.2.0']
+DEPENDENCIES = ['amcrest', 'ffmpeg']
```

Comment:
I think that need not be `ffmpeg`, he don't access to `ffmpeg` service or code here.